edge-modules/SimulatedTemperatureSensor/src/Program.cs

```diff
@@ -224,18 +237,31 @@ static async Task SendEvents(
                     eventMessage.ContentType = "application/json";
                     eventMessage.Properties.Add("sequenceNumber", count.ToString());
                     eventMessage.Properties.Add("batchId", BatchId.ToString());
-                    Console.WriteLine($"\t{DateTime.Now.ToLocalTime()}> Sending message: {count}, Body: [{dataBuffer}]");
-
-                    await moduleClient.SendEventAsync("temperatureOutput", eventMessage);
+                    logger.LogInformation($"Sending message: {count}, Body: [{dataBuffer}]");
+                    try{
```

Comment:
To be consistent with the coding style elsewhere in this file and in the codebase, please put the opening brace for the try statement on its own line. Same for the try statement a few lines down...